src/Core/Cache/ManifestCacheFactory.php

```diff
@@ -70,23 +79,22 @@ public function create($service, array $params = [])
     }
```

Comment:
I can't put a comment on the actual line above - but the exception needs to be updated since there are now _two_ interfaces that it could implement to be valid, not just `CacheInterface`.

Reply:
Updated